pypiserver/config.py

```diff
@@ -719,6 +736,22 @@ def auther(uname: str, pw: str) -> bool:
 
         return auther
 
+    def get_backend(self, arg: str) -> IBackend:
+
+        available_backends: t.Dict[str, BackendFactory] = {
+            "auto": get_file_backend,
+            "simple-dir": SimpleFileBackend,
+            "cached-dir": CachingFileBackend,
+        }
+        # if arg not in available_backends:
+        #     raise argparse.ArgumentTypeError(
+        #         f"Value must be one of {', '.join(available_backends.keys())}"
+        #     )
+
+        backend = available_backends[arg]
+
+        return BackendProxy(backend(self))
```

Comment:
Should we put this in the `_ConfigCommon` so that it's available for the update command too?

Reply:
Yes, that makes sense to me. I think the main reason I didn't do that yet, was because it currently requires `hash_algo` to be part of the same config and that one's on the RunConfig. But if you're fine with moving that to common, then I see no reason to not do that.

Reply:
Yep, now that we need it in both the run and update configs, we might as well move it